content/citation-tags.tsv

```diff
@@ -277,3 +277,18 @@ Zintgraf2017_visualizing	arxiv:1702.04595
 goodfellow2016deep	url:http://www.deeplearningbook.org/
 li2016joint	url:http://www.ijcai.org/Proceedings/16/Papers/403.pdf
 world2004international	url:http://www.who.int/classifications/icd/en/
+ghahramani_protect	arxiv:1707.02476
+uncertainty_types	arxiv:1703.04977
```

Comment:
Have you seen the NIPS 2017 version of this paper?  Can we cite `url:http://papers.nips.cc/paper/7141-what-uncertainties-do-we-need-in-bayesian-deep-learning-for-computer-vision` instead?

Reply:
@alexandari I support @dhimmel's [comment below](https://github.com/greenelab/deep-review/pull/766#issuecomment-357291716). Let's just leave these arXiv references because there is no good way to automatically scrape the citations from the conference websites.  Cross referencing #699 in case I want to do it manually at some point.

Reply:
Sounds good. I will leave the arXiv references.